specification/http/src/main/scripts/org/kaazing/specification/http/rfc7235/framework/forbidden/request.rpt

```diff
@@ -0,0 +1,9 @@
+connect http://localhost:8000/resource
+connected
+
+write method "GET"
+write header "Authorization" "Basic" ${http:loginBase64Encoder("admin:admin")}

```

Comment:
I don't think `write header` can support multiple values. I ran this test and captured the wireshark and I observed that K3PO server will add multiple Authorization header.

```
GET http://localhost:8000/resource HTTP/1.1
Authorization: Basic
Authorization: YWRtaW46YWRtaW4=
```